kcl/kcl.go

```diff
@@ -152,7 +152,7 @@ type KCLProcess struct {
 	recordProcessor RecordProcessor
 
 	nextCheckpointPair SequencePair
-	wasAskedToShutdown bool
+	doGracefulShutdown bool
```

Comment:
what's the difference between the two. Just curious.

Reply:
I felt `doGracefulShutdown` was a clearer name